### DISCUSSION TITLE
How to export model back to Lychee?

### DISCUSSION THREAD

**Discussion by h2ogeek** [sn4k3/UVtools#643] [Q&A] [answered]
Lychee is my main slicer, and I export my CTB files to UVtools.  It seems like UVTools does not offer the option to export to (STL,OBJ, or .3MF). What tool do you recommend to convert from CTB to (STL, .OBJ, .3MF).

Cheers

**Comment by sn4k3** [comment, marked as answer]
UVtools can export layers to mesh (STL) and re-create sliced model from 2D. 

But once sliced and saved, the output file contains only 2D slices of the model. Model original mesh is lost, ie: a perfect shape will now look pixelized. Converting slices to mesh results in a very large file and not suitable to reslice and in some cases will be so large that your program will hang while try to open such.

Same happens with chitubox, you can load CTB back but it's for preview purposes (If you do you will notice that open model looks nothing like the original smooth-wise), you should never re-slice a sliced file which will result in worse model quality as you are slicing a sliced file rendered from 2D.

**What you should do:**
- Never close slicer until you happy with model pos-process (UVtools), so in any case you can fix stuff on slicer and repeat the process.
- In lychee save your scene/project (.lys) so you can load it back with the original information and others like supports as separate objects

**Good pratices:**
1. Always keep STL (my_model1554.stl)
2. Save slicer scene (my_model1554.lys)
3. Slice (my_model1554.ctb)

That way you have everything and not losing any information. If modifications are required you can just open the .lys and edit from there, then reslice...
